### DISCUSSION TITLE
Use of title attribute

### DISCUSSION THREAD

**Discussion by 3phase** [IgniteUI/igniteui-angular#10454] [Q&A]
## Question  
Is there any particular reason (besides #10419) to use the global title attribute throughout the grid, instead of our tooltip component?

**Comment by ChronosSF** [comment]
It was probably discussed at some point in the past  - @rkaraivanov ?

We already discussed that `title` is probably mandatory unless we want to show the tooltip on focus too (afaik it shows on mouseover atm). This is be cause narrators have to read based on keyboard interaction only. If the tooltip can be made to use and replace the default one using the value in the `title` then it is strictly a design decision so should be aimed more towards @StefanIvanov
